web/fido_app/static/scripts/add-credential.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Defines the registration behavior for our app, which relies on the
+ * Web Authentication API to enable passwordless/biometric login (see
+ * https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API).
+ *
+ * It relies on the WebAuthnApp example, provided here:
+ * https://github.com/webauthn-open-source/webauthn-simple-app
+ */
+
+// Find form elements in the DOM
+const addBiometric = document.getElementById("add-biometric")
```

Comment:
Hmmm. I'm curious if there's a better way to do this (to prevent duplicating the registration process code for another route).